src/NetAnalyzers/Core/Microsoft.NetFramework.Analyzers/Helpers/SyntaxNodeHelper.cs

```diff
@@ -123,17 +123,17 @@ public IEnumerable<IMethodSymbol> GetCalleeMethodSymbols(SyntaxNode? node, Seman
             return GetDeclaredSymbol(node, semanticModel) ?? GetReferencedSymbol(node, semanticModel);
         }
 
-        public static ISymbol? GetDeclaredSymbol(SyntaxNode? node, SemanticModel semanticModel)
+        public static ISymbol? GetDeclaredSymbol(SyntaxNode? node, SemanticModel semanticModel) // TODO: Should I add an optional CancellationToken argument here to this public method?
```

Comment:
I don't think we want to spend any energy on analyzers in `Microsoft.NetFramework.Analyzers` project. Can you revert changes in this file and other files in NetFramework projects?